src/cache/redisCache.module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from '@nestjs/common';
+import { CacheModule } from '@nestjs/cache-manager';
+import { ConfigModule } from '@nestjs/config';
+import * as redisStore from 'cache-manager-redis-store';
+import { RedisCacheService } from './redisCache.service';
+
+@Module({
+  imports: [
+    CacheModule.register({
+      imports: [ConfigModule],
+      useFactory: async () => ({
+        store: redisStore,
+        url: 'redis://localhost:6379',
```

Comment:
`url` should be moved to environment variables as it might change with the environment